scripts/e2e.sh

```diff
@@ -33,24 +38,25 @@ $DOCKER run --name $PSQL_CONTAINER -e POSTGRES_PASSWORD=password -p $PSQL_PORT:5
 echo "Wait 5s for postgresql to start..."
 sleep 5
 
+export SIPPY_E2E_DSN="postgresql://postgres:password@localhost:$PSQL_PORT/postgres"
```

Comment:
Why can't this just be SIPPY_DATABSE_DSN?

Reply:
This e2e cleans out the regressions table after each test. I wanted to be exceedingly careful someone doesn't accidentally run it against prod, so I thought it made sense to make it a very explicit e2e var that isn't going to accidentally be set in any situation I could come up with. WDYT?